### PR TITLE
Repository branch

### PR DESCRIPTION
**Pull request by Luis-GA**

### CODE REVIEW

File: mano/mano.conf [hunk 1]
```diff
@@ -1,24 +1,25 @@
 [NFVO]
-TYPE=<type of NFVO. Currently only 'OSM' is supported>
-IP=<IP address of your NFVO>
-USER=<user name with admin rights within the NFVO>
-PASSWORD=<Password to the previous user>
+TYPE=OSM
```
Comment (jmelian) [inline mano/mano.conf]: Keep instructions instead of actual values

File: mano/mano.conf [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+IP=192.168.33.20
+USER=admin
+PASSWORD=admin
+VIM_ACCOUNT=MalagaOS_testEnv
```
Comment (jmelian) [inline mano/mano.conf]: Keep instructions instead of actual values

File: mano/requirements.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ Flask==1.0.2
 gevent==1.4.0
 pymongo==3.8.0
 Flask-RESTful==0.3.7
-PyYAML==5.1
+PyYAML
```
Comment (jmelian) [inline mano/requirements.txt]: Add version:
PyYAML==5.1

File: README.md [hunk 4]
```diff
@@ -126,22 +110,12 @@ Once edited properly, the configuration will be applied and the containers built
...
-    HOST=validator
-    port=5100
-    path=/
     [mano]
     PROTOCOL=http
     HOST=mano
     PROTOCOL=5001
     PATH=/
     
-    [elcm]
```
Comment (jmelian) [inline README.md:140]: Keep this lines as part of the example

File: README.md [hunk 5]
```diff
@@ -19,16 +19,6 @@ The available features will depend on the features exposed by each dispatched mo
 
 ##### MANO enabler
 - Validation service: Validate VNFD or NSD as a standalone service
-- Onboard VNFD
```
Comment (jmelian) [inline README.md:22]: Why are you removing all these features? Are they no longer valid?

File: dispatcher.conf [hunk 6]
```diff
@@ -1,13 +1,3 @@
-[component_name]
```
Comment (jmelian) [inline dispatcher.conf:1]: Keep instructions instead of actual values

**Review by jmelian — CHANGES_REQUESTED** [review]
Please, address the comments and send back the pull request

File: README.md (hunk 5 above)
Comment (Luis-GA) [reply README.md:22]: These are not the currently functions of the mano component

File: README.md (hunk 4 above)
Comment (Luis-GA) [reply README.md:140]: I think that could be confused

File: dispatcher.conf (hunk 6 above)
Comment (Luis-GA) [reply dispatcher.conf:1]: I think these are deprecated

File: mano/mano.conf (hunk 1 above)
Comment (Luis-GA) [reply mano/mano.conf]: ok!

File: mano/requirements.txt (hunk 3 above)
Comment (Luis-GA) [reply mano/requirements.txt]: ok!

File: mano/mano.py [hunk 7]
```diff
@@ -1,384 +1,270 @@
...
+def get_vims():
+    """
+    Output: 200 code and list of current vims
+    """
+    logger.info("Retrieving VIMs list")
+    aux_list = []
+    try:
+        for vim in conf["VIM"]:
+            new_vim = {}
+            new_vim["name"] = conf["VIM"][vim]["NAME"]
```
Comment (jmelian) [inline mano/mano.py]: new_vim["name"] = vim

File: README.md [hunk 8]
```diff
@@ -19,16 +19,6 @@ The available features will depend on the features exposed by each dispatched mo
...
-- List VNFDs
-- Retrieve single VNFD
-- Delete VNFD
-- Update VNFD
-- Onboard NSD
-- List NSDs
-- Retrieve single NSD
-- Delete NSD
-- Update NSD
```
Comment (jmelian) [inline README.md:31]: These features don't disappear, If they don't belong to the MANO, they must be somewhere else

**Review by jmelian — CHANGES_REQUESTED** [review]

**Pull request merged**